css/fonts.css

```diff
@@ -0,0 +1,35 @@
+@font-face {
+  font-family: 'Montserrat';
+  src:
+    url('../fonts/montserrat-bold-webfont.woff2') format('woff2'),
+        url('../fonts/montserrat-bold-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/finalproject/pullRequest?prid=5950407)